app/models/story.rb

```diff
@@ -117,6 +120,14 @@ def tags=(ts)
     self.user_tags = ts.uniq.sort.map { |t| UserTag.new(name: t) }
   end
 
+  def transcript
```

Comment:
It is another request from the AMPERs folks for their network integration. I decided just to put it in with the other networks related changes for them.